hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterMountTableCacheRefresh.java

```diff
@@ -234,7 +234,7 @@ public void testCachedRouterClientBehaviourAfterRouterStoped()
       assertEquals(srcPath, mountTableResult.getSourcePath());
     }
 
-    // Lets stop one router
+    // Let's stop one router
```

Comment:
revert this, guess Let's is let us and lets is to permit or to allow, it is ok in this context